project/pom.xml

```diff
@@ -1280,7 +1280,10 @@
             <groupId>org.gaul</groupId>
             <artifactId>modernizer-maven-plugin</artifactId>
             <configuration>
+              <!-- Fix fiolations before uncommenting
```

Comment:
@nacx Is the plan to address this as part of the Java 8 support?

Reply:
Yes, definitely. I wanted to enter this change because this was blocking https://github.com/jclouds/jclouds-labs/pull/439, but we definitely need to get those warnings fixed.